install/kubernetes/helm/istio/charts/kiali/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: v1
 description: Kiali is an open source project for service mesh observability, refer to https://github.com/kiali/kiali for detail.
 name: kiali
-version: 0.3.1.Alpha
+version: 0.3.0-alpha
```

Comment:
I would recommend just using version = 0.8.0 since Kiali is integrated.  appVersion can then be used to specify the version as is done on line 5.

Reply:
Should not we use `version: 0.8.0-1` for Kiali as well?